Challange-1-iOS/Challange-1-iOS/ViewControllers/AppleReposViewController.swift

```diff
@@ -7,14 +7,158 @@
 
 import UIKit
 
+struct ConstantsAppleRepos {
+    static let rowHeight: CGFloat = 50
+}
+
 class AppleReposViewController: UIViewController, Coordinating {
+    private var tableView: UITableView
+    private var appleRepos: [AppleRepos] = []
+    private var mockedAppleReposDataSource: MockedAppleReposDataSource = .init()
+    
+    var appleReposService: AppleReposService?
+    
+    private var itemsPerPage:Int = 10
+    private var pageNumber: Int = 1
+    
+    private var addedToView: Bool = false
+    private var isEnd: Bool = false
+    
+    
     var coordinator: Coordinator?
     
-
+    init() {
+        tableView = .init(frame: .zero)
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        setUpViews()
+        addViewsToSuperview()
+        setUpConstraints()
+    }
+    
+    private func setUpViews() {
+        setUpTableView()
+    }
+    
+    private func addViewsToSuperview() {
+        view.addSubview(tableView)
+    }
+    
+    private func setUpConstraints() {
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: view.topAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            tableView.leftAnchor.constraint(equalTo: view.leftAnchor),
+            tableView.rightAnchor.constraint(equalTo: view.rightAnchor)
+        ])
+    }
+    
+    private func setUpTableView() {
         title = "Apple Repos"
-        view.backgroundColor = .appColor(name: .surface)
+        
+        tableView.rowHeight = ConstantsAppleRepos.rowHeight
```

Comment:
In your approach, you're using a static number. In general, we don't do that, it's better to make it dynamic, i.e: the cell's size should adapt based on the content size. How can you do that?

Reply:
I think remove the line 71 is enough

Reply:
I removed the line 71 and set the lineBreakMode of the text in the set to .byWordWrapping